pkg/frontend/mysql_cmd_executor.go

```diff
@@ -910,20 +911,20 @@ func (mce *MysqlCmdExecutor) handleCmdFieldList(tableName string) error {
 
 			names := db.Relations(txnHandler.GetTxn().GetCtx())
 			for _, name := range names {
-				table, err := db.Relation(name, txnHandler.GetTxn().GetCtx())
+				_, err := db.Relation(name, txnHandler.GetTxn().GetCtx())
 				if err != nil {
 					return err
 				}
-				defs := table.TableDefs(txnHandler.GetTxn().GetCtx())
-				for _, def := range defs {
-					if attr, ok := def.(*engine.AttributeDef); ok {
-						// Fixme: attrs here is never used.
-						attrs = append(attrs, &engineColumnInfo{
-							name: attr.Attr.Name,
-							typ:  attr.Attr.Type,
-						})
-					}
-				}
+				// Fixme: attrs here is never used.
```

Comment:
Will it ever be used?   Should you just delete the code?

Reply:
I will confirm with @daviszhen

Reply:
just retain the code. If it has errors i will fix it in my next pr.

Reply:
get it